apps/rpc-server/src/index.ts

```diff
@@ -59,10 +57,10 @@ export class RPCServer {
         timeout: this.timeout,
         discoveryPlatformApiEndpoint: this.discoveryPlatformApiEndpoint,
       },
-      async (clientId, counter) => {
+      async (clientId: any, counter: any) => {
```

Comment:
shouldn't these already have a type from `RPChSDK`?

Reply:
removed this type, seems like an initial eslint error but yes they are infered from RPChSDK